sled-agent/src/sim/http_entrypoints.rs

```diff
@@ -438,10 +438,9 @@ impl SledAgentApi for SledAgentSimImpl {
         ))
     }
 
-    async fn support_bundle_get(
+    async fn support_bundle_download(
```

Comment:
I'm re-working the simulated API in https://github.com/oxidecomputer/omicron/pull/7063 -- this is mostly a placeholder for now